build/goreleaser/linux/centos_7_amd64.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-amd64
       - linux-service-amd64
     package_name: newrelic-infra
-    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el7.{{ .Arch }}"
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el7.{{ .Arch }}{{ .Env.FIPS }}"
```

Comment:
Based on the recommendation by a team member, we want to move the FIPS suffix and place it after the app name newrelic-infra-fips..., this is to follow the pattern packages normally follow, so this would be:

`"newrelic-infra{{ .Env.FIPS }}-{{ .Env.TAG }}-1.el7.{{ .Arch }}"`

Reply:
I swapped this, should be good now